pkgs/cupertino_http/lib/src/cupertino_api.dart

```diff
@@ -199,7 +211,7 @@ class URLSessionConfiguration
     }
     final d = objc.NSMutableDictionary.alloc().init();
     headers.forEach((key, value) {
-      d.setObject_forKey_(value.toNSString(), key.toNSString());
+      d.setObject(value.toNSString(), forKey: key.toNSString());
```

Comment:
`NSMutableDictionary` is a `Map` now, so `d[key.toNSString()] = value.toNSString()` should work. Actually, you can probably use the built in conversion function and get rid of this whole loop: `final d = toObjCObject(headers) as NSMutableDictionary;`